### ISSUE TITLE
Update Manager should describe the standard repository (or repositories)

### ISSUE THREAD

**Issue by grahamperrin** [trueos/sysadm-ui-qt#13]
If no custom repository is entered by the user: 
- the GUI should describe, in plain English/translation, the (iXsystems-provided) repository that is the default (if iXsystems will not maintain more than one), or selected.

Expect some users who were familiar with PC-BSD to wonder whether the TrueOS default will be closer, in principle, to _edge_ than to _production_. The GUI can offer the simplest possible explanation (and then the user may seek additional information from the handbook, a blog post, release notes or whatever).

**Comment by grahamperrin** [comment]
Today I reduced the SysAdmin window to its smallest and realised that there's not always room for a description. Still, I think that a description will be nice.

**Comment by q5sys** [comment]
Housecleaning time.  Many changes have occurred so I'm closing old tickets. If the issue still remains we can reopen.
